src/views/Portfolio.vue

```diff
@@ -169,92 +188,64 @@
                         </div>
                     </td>
                     <td>
-                        <router-link :to="{ name: 'Synths', params: { synth: synth.id } }">
+                        <router-link :to="{ name: 'Synths', params: { synth: synth.symbol } }">
                             <img src="@/assets/images/arrow-right.svg" class="cursor-pointer" />
                         </router-link>
                     </td>
                 </tr>
+                <tr v-if="loading" class="border-b bg-main basic-hover">
+                    <td class="px-4 py-2 font-semibold flex">
+                        <div class="rounded-full animate-pulse bg-white w-6 h-6" />
+                        <div class="animate-pulse bg-white pl-2 w-full">.</div>
+                    </td>
+                    <td><div class="animate-pulse bg-white">.</div></td>
+                    <td><div class="animate-pulse bg-white">.</div></td>
+                    <td>
+                        <div class="animate-pulse bg-white">
+                            <span class="text-white">.</span>
+                        </div>
+                    </td>
+                    <td>
+                        <div class="rounded-full animate-pulse bg-white w-6 h-6" />
+                    </td>
+                </tr>            
```

Comment:
to convert it into a component, then use it up and here

Reply:
This would mean we would add both loaders (for mobile and desktop) in separate components.

Reply:
both should be in the same component, on the mobile it would just show what it supposed to